tools/taskcluster/tasks.py

```diff
@@ -0,0 +1,143 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import datetime
+import json
+import os
+
+import taskcluster
+
+
+def artifact(artifact_type, absolute_path):
+    return {
+        'type': artifact_type,
+        'path': absolute_path,
+        'expires': taskcluster.stringDate(taskcluster.fromNow('1 week'))
```

Comment:
Nit: I guess we want to extend the duration to 1 year once the code reaches master. 1 week seems to short.

Reply:
Can do! I just copied this from the existing `.taskcluster.yml`, but it makes sense to extend it